src/InstagramScraper/Model/Account.php

```diff
@@ -464,6 +478,9 @@ protected function initPropertiesCustom($value, $prop, $array)
             case 'edge_follow':
                 $this->followsCount = !empty($array[$prop]['count']) ? (int)$array[$prop]['count'] : 0;
                 break;
+            case 'follower_count':
```

Comment:
If you confirmed that `follower_count` exists then could you please change 
`$this->followersCount = $value;` to `$this->followedByCount = $value`